rust/main/chains/hyperlane-ethereum/src/tx.rs

```diff
@@ -163,10 +165,10 @@ where
     }
 
     let eip1559_fee_result = match cached_eip1559_fee {
-        Some(fee) => Ok(fee),
-        None => estimate_eip1559_fees(provider.clone(), None, &latest_block, domain, &tx.tx).await,
+        Some(fee) => Ok((fee, latest_block)),
+        None => estimate_eip1559_fees(provider.clone(), None, domain, &tx.tx).await,
```

Comment:
We request latest block two times in `fill_tx_gas_params`, but since the request of the latest block inside `estimate_eip1559_fees` happens in parallel to requesting fee history, the overall latentcy is not impacted for regular chains. It is impacted for zksync since we don't request fee history there.

Reply:
It is ok. Discussed in real life.